src/controllers/auth.controller.js

```diff
@@ -17,14 +17,16 @@ const AuthHandler = async (req, res) => {
 			const accessToken = jwt.sign(
 				{ id: user.id, role: user.role },
 				process.env.ACCESS_TOKEN_SECRET,
-				{ expiresIn: '600s' }
+				{ expiresIn: '120s' }
```

Comment:
The duration has been reduced since on the front-end all requests will be intercepted to fetch a new refresh token